besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1284,6 +1284,12 @@ static class TxPoolOptionGroup {
           "Specifies the static node file containing the static nodes for this node to connect to")
   private final Path staticNodesFile = null;
 
+  @CommandLine.Option(
+      names = {"--rpc-max-logs-range"},
+      description =
+          "Specifies the maximum number of blocks to retrieve logs from via RPC. Value must be > 0")
+  private final Long rpcMaxLogsRange = 0L;
```

Comment:
would it make it less of a breaking change to have the default be a bigger number?

Reply:
0 = option switched off in this case. It needs to be > 1 to enable the range check.

Reply:
I think we need a non-zero default and doc it as a breaking change (if your workflow relies on more than the default)